VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/join-private-consultation/join-private-consultation.component.scss

```diff
@@ -32,8 +33,14 @@
   padding: 20px;
   border: none;
   border-radius: 5px;
-  width: 50%;
   max-width: 700px;
+
+  @media only screen and (min-width: $talbet-landscape-width) {
+    width: 50%;
+  }
+  @media only screen and (max-width: $talbet-landscape-width) {
+    width: 82%;
+  }
```

Comment:
There's still the issue where the min-width and max-width overlap. In a mobile first application we would set the default value we want to see and then use the breakpoints to override based only on the lower bound like the following:

```
   width: 82%;
   
  @media only screen and (min-width: $talbet-landscape-width) {
    width: 50%;
  }
``` 

But as everything here has been done desktop first it's probably more accurate to do:
```
   width: 50%;
   
    @media only screen and (max-width: $talbet-landscape-width) {
    width: 82%;
  }
``` 

I'm not really sure which of the two we should use here.